docs/tutorials/t2ramsey_tutorial.ipynb

```diff
@@ -0,0 +1,416 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "from qiskit_experiments.characterization import T2RamseyAnalysis\n",
+    "# The user can provide initial estimates for the parameters to help the analysis process\n",
+    "user_p0={\n",
+    "    \"A\": 0.5,\n",
+    "    \"t2ramsey\": 85.0,\n",
+    "    \"f\": 0.021,\n",
+    "    \"phi\": 0,\n",
+    "    \"B\": 0.5\n",
+    "        }\n",
+    "exp_with_p0 = T2Ramsey(qubit, delays, unit=unit)\n",
+    "exp_with_p0.set_analysis_options(user_p0=user_p0, plot=True)\n",
+    "expdata_with_p0 = exp_with_p0.run(backend=backend, shots=2000)\n",
+    "print(expdata_with_p0)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The units can be changed, but the output in the result is always given in seconds. The units in the backend must be adjusted accordingly."
```

Comment:
> The units in the backend must be adjusted accordingly.

This is just for the mock backend right ? Need to make explicit since real backends are different

Reply:
I'd drop this line, considering that it's not a tutorial about how to use the mock backend

Reply:
Pedagogically, I think the mock backend is useful, when one wants a quick learning TATs. I think even a dedicated sub-section for the mock backend is in place (documenting how to tweak it, nothing beyond what is already in the tutorial). This way there is a clean separation between how to use the experiment, and some dedicated reference for the mock backend, allowing users to focus on what they need easily

Reply:
This makes sense. However if the backend changes to Ethan's soon then it may not be worth the effort.

Reply:
This is all probably temporary, so not worth investing too much effort here. The reason I mention the mockjob is because I must import it, and I don't like writing a line that cannot be understood as is.